### PR TITLE
Fix verify intervals

### PR DESCRIPTION
**Pull request by dhilpipre**

### CODE REVIEW

**Review by dhilpipre — COMMENTED** [review]
everything is fine

**Review by dhilpipre — COMMENTED** [review]
looks good

File: camel-kafka-3.18.3/build.gradle
Comment (dhilpipre) [inline camel-kafka-3.18.3/build.gradle:1]: this is a valid change

File: camel-kafka-3.21.4/build.gradle
Comment (dhilpipre) [inline camel-kafka-3.21.4/build.gradle:1]: this is a valid change

File: camel-kafka-3.22.0/build.gradle
Comment (dhilpipre) [inline camel-kafka-3.22.0/build.gradle:1]: this is a valid change

File: camel-kafka-3.18.3/build.gradle
Comment (gsidhwani-nr) [reply camel-kafka-3.18.3/build.gradle:1]: Looks good.
Approved.

**Review by gsidhwani-nr — APPROVED** [review]
Looks good.

**Pull request merged**